src/main/java/org/example/objects/User.java

```diff
@@ -1,30 +1,36 @@
 package org.example.objects;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import org.example.fuzzy_matcher.domain.Document;
 import org.example.fuzzy_matcher.domain.Element;
 import org.example.fuzzy_matcher.domain.ElementType;
 
 import java.util.List;
 
+@JsonIgnoreProperties(ignoreUnknown = true)
 public class User {
   private int id;
   private String name;
   private int age;
-  private List<String> favoriteGenres;
-  private List<String> favoriteArtists;
-  private List<String> favoriteSongs;
-  private List<String> recentlyPlayed;
-
-  public User(int id, String name, int age, List<String> favoriteGenres,
-              List<String> favoriteArtists, List<String> favoriteSongs,
-              List<String> recentlyPlayed) {
+  private List<String> favorite_genres;
```

Comment:
@SebasBarra02 Can you explain me why the change of the variable convention name? Is it was really necessary?
Let me known that but you should maintain the one we have before.